core/domain/exp_domain.py

```diff
@@ -779,8 +780,8 @@ def from_dict(
         exploration.created_on = exploration_created_on
         exploration.last_updated = exploration_last_updated
 
-        exploration.proto_size_in_bytes = (
-            exploration_dict['proto_size_in_bytes'])
+        exploration._cached_android_proto_size_in_bytes = (
```

Comment:
Private variables are internal and should only be accessed from within their class. You should therefore probably only set this variable in an init() method in the Exploration class. Is that possible?

Reply:
Updated